src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -28,7 +28,21 @@ export const TiffLoader = {
     const tiffFiles = files.flat().filter(isTiff);
     if (tiffFiles.length === 0) throw new Error('No Files found');
     if (tiffFiles[0]) await fsa.head(tiffFiles[0]);
-    return await Promise.all(tiffFiles.map((f: string) => new CogTiff(fsa.source(f)).init(true)));
+    const ret = await Promise.allSettled(
+      tiffFiles.map((f: string) => {
```

Comment:
Should `f` be something like `location`, for clarity? It looks like `tiffFiles` is actually a list of file *locations*, rather than files.

Reply:
This was the pre-existing code which was re-jigged a little,  I guess I could rename all of theses variables but it would could add a bunch of noise to the pr.